### PR TITLE
Update to log help text

### PR DESCRIPTION
**Pull request by shavidissa**

### CODE REVIEW

**Review by Margarita-Staneva — APPROVED** [review]
Looks good to me

**Comment by shavidissa** [comment]
We need to add the info about view log alerts to the alert browser help panel. The current URL structure does not support two help panels. @Margarita-Staneva can you take a look, please? :)

File: alerts_intro.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
-To disable checking alert conditions, you can snooze alerts for a fixed time window or put a set of alerts or sources in a custom [maintenance window](https://docs.wavefront.com/maintenance_windows_managing.html#creating-a-maintenance-window).
+**Manage Metrics Alerts**
+ 
+ When a metrics alert changes state, our service sends notifications to the targets your define, such as email addresses, pager services, and real-time communication channels.
+
+To pause the checking alert conditions, you can snooze alerts for a fixed time window or put a set of alerts or sources in a custom [maintenance window](https://docs.wavefront.com/maintenance_windows_managing.html#creating-a-maintenance-window).
+
+**Managing Logs Alerts**
+
+You can create logs alerts and see the firing events of a logs alert. If you have Logs permissions, you see the **Logs Alert** tab, which lists the logs alerts you create. If you have Logs and Alerts permissions, you can create and manage logs alerts.
```
Comment (Margarita-Staneva) [inline alerts_intro.md:13]: If you have the **Logs** permission, you see the **Logs Alert** tab, which lists the logs alerts you create. If you have the **Logs** and **Alerts** permissions, you can create and manage logs alerts.

**Review by Margarita-Staneva — COMMENTED** [review]
I have some comments, please take a look.

**Review by Margarita-Staneva — APPROVED** [review]
lgtm

**Pull request merged**